x-pack/plugins/ml/public/maps/util.ts

```diff
@@ -23,6 +23,33 @@ export const ML_ANOMALY_LAYERS = {
 } as const;
 
 export type MlAnomalyLayersType = typeof ML_ANOMALY_LAYERS[keyof typeof ML_ANOMALY_LAYERS];
+const INFLUENCER_LIMIT = 3;
+const INFLUENCER_MAX_VALUES = 3;
+
+function getInfluencersHtmlString(
+  influencers: Array<{ influencer_field_name: string; influencer_field_values: string[] }>,
+  splitFields: string[]
+) {
+  let htmlString = '<ul>';
+  for (let i = 0; i < influencers.length; i++) {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    const { influencer_field_name, influencer_field_values } = influencers[i];
+    // Skip if there are no values or it's a partition field
+    if (!influencer_field_values.length || splitFields.includes(influencer_field_name)) continue;
+
+    const fieldValuesString = influencer_field_values.slice(0, INFLUENCER_MAX_VALUES).join(', ');
+
+    htmlString += `<li>${influencer_field_name}: ${fieldValuesString}</li>`;
+
+    // Only show top two influencers
```

Comment:
super duper nit: since `INFLUENCER_LIMIT` is a constant and can potentially be changed in the future, I think this comment should not mention "two" specifically.

Reply:
Good point - removed in 1dd59765c032b18dd9de60c53f9a1c24e86cf5d4